vendor/ocamlformat_support/queue.mli

```diff
@@ -0,0 +1,108 @@
+(**************************************************************************)
```

Comment:
Why do you need to vendor `Queue` and `Stack` ?

Reply:
It was needed when we were relying on 4.07.